yb-voyager/cmd/liveMigrationReportCommand.go

```diff
@@ -71,7 +71,8 @@ var liveMigrationReportCmd = &cobra.Command{
 type rowData struct {
 	TableName        string
 	DBType           string
-	SnapshotRowCount int64
+	ExportedRows     int64
```

Comment:
I would suggest:
1. Keep these variables consistent with what is displayed on the UI. So change InsertsIn->ImporterInserts, etc. Makes it easy to correlate code with the UI! 
2. Shall we call it "EXPORTED SNAPSHOT ROWS" and "IMPORTED SNAPSHOT ROWS"?